packages/augur-core/source/contracts/GnosisSafe.sol

```diff
@@ -627,7 +698,32 @@ contract GnosisSafe is MasterCopy, BaseSafe, SignatureDecoder, SecuredTokenTrans
         bytes calldata signatures
     )
         external
-        returns (bool success)
+        returns (bool)
+    {
+        bytes32 txHash = checkTransaction(
+            to, value, data, operation, // Transaction info
+            safeTxGas, baseGas, gasPrice, gasToken, refundReceiver, // Payment info
+            signatures
+        );
+        //require(gasleft() >= safeTxGas, "Not enough gas to execute safe transaction");
```

Comment:
Is this intended to be commented-out?

Reply:
Yea. There is a bug with ganache that will cause it to fail otherwise